### DISCUSSION TITLE
Use dark mode-themed code blocks on dark mode [Edge Docs]

### DISCUSSION THREAD

**Discussion by MatheusRich** [rails/rails#51378] [Docs]
I believe it's more natural to keep the code blocks theme matching. It can cause unnecessary contrast changes when we put a light-themed code block on a page with mostly dark backgrounds.

Here's what  the guides currently look:

<img width="1222" alt="image" src="https://github.com/rails/rails/assets/20938712/52feade3-af4c-4226-8b57-3eeaa6f0c163">

Here's what the guides would look like with a Monokai theme:

<img width="1248" alt="image" src="https://github.com/rails/rails/assets/20938712/8e6c0392-4f21-4eb2-9d4e-f6e0ad525031">

## Prior work

Both [Python](https://docs.python.org/3/library/stdtypes.html#int.bit_length) and [Elixir docs](https://hexdocs.pm/elixir/1.16.2/Kernel.html#module-structural-comparison) do similar things (i.e., dark mode code blocks on dark mode theme).

**Comment by AmandaPerino** [comment]
@MatheusRich And this one - please close this issue since it is addressed in #51372. Thank you so much for your input!
